test/e2e/seamless_migration_test.go

```diff
@@ -0,0 +1,116 @@
+			propagationPolicy = helper.NewPropagationPolicy(deployment.Namespace, deployment.Name, []policyv1alpha1.ResourceSelector{
+				{
+					APIVersion: deployment.APIVersion,
+					Kind:       deployment.Kind,
+					Name:       deployment.Name,
+				},
+			}, policyv1alpha1.Placement{
+				ClusterAffinity: &policyv1alpha1.ClusterAffinity{ClusterNames: []string{"member1"}},
+			})
+			bindingName = names.GenerateBindingName(deployment.Kind, deployment.Name)
+		})
+
+		ginkgo.AfterEach(func() {
+			framework.RemoveDeployment(kubeClient, deployment.Namespace, deployment.Name)
+			framework.RemovePropagationPolicy(karmadaClient, propagationPolicy.Namespace, propagationPolicy.Name)
+		})
+
+		ginkgo.It("Test migrate a Deployment from member cluster", func() {
+			// Step 1, Create Deployment in member1 cluster
+			ginkgo.By("Creating Deployment in member1 cluster", func() {
+				framework.CreateDeployment(member1Client, deployment)
+			})
+
+			// Step 2, Create Deployment in karmada control plane
+			ginkgo.By("Creating Deployment in karmada control plane", func() {
+				framework.CreateDeployment(kubeClient, deployment)
+			})
+
+			// Step 3, Create PropagationPolicy in karmada control plane without conflictResolution field
+			ginkgo.By("Creating PropagationPolicy in karmada control plane without conflictResolution field", func() {
+				framework.CreatePropagationPolicy(karmadaClient, propagationPolicy)
+			})
+
+			// Step 4, Verify ResourceBinding got unHealthy for resource already exist
+			ginkgo.By(fmt.Sprintf("Verify ResourceBinding %s got unHealthy for resource already exist", bindingName), func() {
+				gomega.Eventually(func() bool {
+					binding, err := karmadaClient.WorkV1alpha2().ResourceBindings(deployment.Namespace).Get(context.TODO(), bindingName, metav1.GetOptions{})
+					if err != nil && apierrors.IsNotFound(err) {
+						return false
+					}
+					gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+
+					items := binding.Status.AggregatedStatus
+					return len(items) > 0 && items[0].Applied == false && items[0].Health != workv1alpha2.ResourceHealthy
+				}, pollTimeout, pollInterval).Should(gomega.Equal(true))
+			})
+
+			// Step 5, Update PropagationPolicy in karmada control plane with conflictResolution=Overwrite
+			ginkgo.By(fmt.Sprintf("Update PropagationPolicy %s in karmada control plane with conflictResolution=Overwrite", propagationPolicy.Name), func() {
+				propagationPolicy.Spec.ConflictResolution = policyv1alpha1.ConflictOverwrite
+				framework.UpdatePropagationPolicyWithSpec(karmadaClient, propagationPolicy.Namespace, propagationPolicy.Name, propagationPolicy.Spec)
+			})
+
+			// Step 6, Verify Deployment Replicas all ready and ResourceBinding got Healthy for overwriting conflict resource
+			ginkgo.By(fmt.Sprintf("Verify Deployment Replicas all ready and ResourceBinding %s got Healthy for overwriting conflict resource", bindingName), func() {
+				gomega.Eventually(func() bool {
```

Comment:
may need add some logs like `klog.Infof("Waiting for namespace present on cluster(%s)", clusterName)`